data/EdlienTheory.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "Edlien Theory",
+  "type": "community",
+  "bio": "I've brought you a bargain community member, become a open source legend in trade of never using non-inclusive language",
+  "avatar": "https://github.com/EdlienTheory.png",
```

Comment:
As of our latest update, the `avatar` key is no longer needed.
```suggestion
```

Reply:
Thanks for reminding!